src/converters/monitor.ts

```diff
@@ -140,6 +140,15 @@ export class NumericCharacteristicMonitor extends BaseCharacteristicMonitor {
     }
     const percentage = (input - this.input_min) / (this.input_max - this.input_min);
 
-    return out_minimum + percentage * (out_maximum - out_minimum);
+    const result = out_minimum + percentage * (out_maximum - out_minimum);
+    if (
+      result < 1 &&
+      result > 0 &&
+      actualCharacteristic !== undefined &&
+      actualCharacteristic.UUID === '00000008-0000-1000-8000-0026BB765291'
```

Comment:
I'm pretty sure there's a define for this in the HAP interface somewhere.
Would prefer to use that, instead of this "magic value".

Reply:
But, I think it's even better if we just add a boolean property to this class that indicates of it should round up if below 1.

By default this would be false, but when we instantiate it for the Brightness characteristic, we can set it to true.

That way, we don't need to know about the characteristic itself here and we can easily reuse this logic elsewhere if needed.
(I can imagine other characteristics that have a percentage as unit might suffer from the same bug.)